backend/score.py

```diff
@@ -129,11 +130,25 @@ async def extract_knowledge_graph_from_file(
     
 
 @app.get("/sources_list")
-async def get_source_list(uri=Form(),database=Form(),userName=Form(),password=Form()):
+async def get_source_list(uri:str,
+                          database:str,
+                          userName:str,
+                          password:str):
     """
     Calls 'get_source_list_from_graph' which returns list of sources which alreday exist in databse
     """
-    result = await asyncio.to_thread(get_source_list_from_graph,uri,database,userName,password)
+    # base64_bytes = base64.b64encode(password.encode("utf-8"))
+    # base64_encoded_string = base64_bytes.decode("utf-8")
+    # print("encoded password=",base64_encoded_string)
+   
+    #base64_bytes = password.encode("ascii")
```

Comment:
this should probably handled in one place not in every function

Reply:
Created separate function for decode password